wp-cldr.php

```diff
@@ -10,6 +10,138 @@
  * Domain Path: /languages/
  * License:     GPLv2 or later
  * License URI: https://www.gnu.org/licenses/gpl-2.0.html
+ *
+ * @package    wp-cldr
  */
 
 require_once plugin_dir_path( __FILE__ ) . 'class.wp-cldr.php';
+require_once( ABSPATH . 'wp-admin/includes/translation-install.php' );
+
+add_action( 'admin_menu', 'wp_cldr_menu' );
+
+/**
+ * Gets the wp-admin menu info.
+ */
+function wp_cldr_menu() {
+	add_options_page(
+		'WP CLDR Options',
+		'WP CLDR',
+		'manage_options',
+		'wp-cldr',
+		'wp_cldr_settings'
+	);
+}
+
+/**
+ * Gets the settings and examples page for the plugin.
+ */
+function wp_cldr_settings() {
+
+	if ( isset( $_GET['locale'] ) && 1 < strlen( sanitize_text_field( wp_unslash( $_GET['locale'] ) ) ) ) {
+		$locale = sanitize_text_field( wp_unslash( $_GET['locale'] ) );
```

Comment:
We don't really have a single whitelist of locales in large part because there's a wide range of formats and especially in .org users might add more. Because of the mapping in `get_cldr_locale()`  and the fallback logic in `initialize_locale_bucket()` the plugin would support them, but we wouldn't necessarily know the locale code ahead of time.

So my gut is to leave this way for now, and maybe down the road try to create some kind of validation function that would take a string and determine whether it's a valid locale code and whether it resolves to one of the included CLDR files.

how's that sound?